code/game/objects/structures/reflector.dm

```diff
@@ -1,44 +1,124 @@
 /obj/structure/reflector
-	name = "reflector frame"
-	icon = 'icons/obj/stock_parts.dmi'
-	icon_state = "box_0"
+	name = "reflector base"
+	icon = 'icons/obj/reflector.dmi'
+	icon_state = "reflector_map"
 	desc = "A frame to create a reflector.\n<span class='notice'>Use <b>5</b> sheets of <b>glass</b> to create a 1 way reflector.\nUse <b>10</b> sheets of <b>reinforced glass</b> to create a 2 way reflector.\nUse <b>1 diamond</b> to create a reflector cube.</span>"
-	anchored = 0
-	density = 1
+	anchored = FALSE
+	density = FALSE
```

Comment:
`density = 1` changed to `density = FALSE` should this be `density = TRUE`?

Reply:
This is intentional, the density on the frame is false and it can be walked over until the reflector is properly completed and constructed which it then becomes dense